.ci/cico_build_deploy_test_rhche.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+# ERROR CODES:
+# 1 - missing file
+# 2 - missing credentials
+# 3 - OpenShift login failed
+# 4 - command execution failed
+
+set -x
+set -e
+set +o nounset
+
+/usr/sbin/setenforce 0
+
+export BASEDIR=$(pwd)
+export DEV_CLUSTER_URL=https://devtools-dev.ext.devshift.net:8443/
+export OC_VERSION=3.9.33
+export TARGET=${TARGET:-"rhel"}
```

Comment:
Are you sure you want to have `rhel` be the default target ?
What's going to happen if we try to run the script locally on a RHEL laptop ?

Reply:
@ScrewTSW What do you think about that?